adguard/Dockerfile

```diff
@@ -8,12 +8,12 @@ ARG BUILD_ARCH=amd64
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup base
-ARG ADGUARD_HOME_VERSION="v0.107.65"
+ARG ADGUARD_HOME_VERSION="v0.107.64"
 # hadolint ignore=DL3003,SC2155
 RUN \
     apk add --no-cache \
         nginx=1.28.0-r3 \
-        yq-go=4.46.1-r1 \
+        yq-go=4.46.1-r2 \
```

Comment:
As previously mentioned, this is not related to this pull request and thus should not be in this one.

../Frenck